contracts/src/v0.7/dev/KeeperRegistrarDev.sol

```diff
@@ -87,16 +86,15 @@ contract KeeperRegistrarDev is TypeAndVersionInterface, ConfirmedOwner {
   event AutoApproveAllowedSenderSet(address indexed senderAddress, bool allowed);
 
   event ConfigChanged(
-    AutoApproveType configType,
-    uint32 windowSizeInBlocks,
-    uint16 allowedPerWindow,
+    AutoApproveType autoApproveConfigType,
+    uint32 autoApproveMaxAllowed,
     address keeperRegistry,
-    uint256 minLINKJuels
+    uint96 minLINKJuels
   );
 
-  constructor(address LINKAddress, uint256 minimumLINKJuels) ConfirmedOwner(msg.sender) {
+  constructor(address LINKAddress, uint96 minLINKJuels) ConfirmedOwner(msg.sender) {
     LINK = LinkTokenInterface(LINKAddress);
-    s_minLINKJuels = minimumLINKJuels;
+    s_config.minLINKJuels = minLINKJuels;
```

Comment:
yeah I'm with you @infiloop2 , let's remove `minLINKJuels` from the constructor

Reply:
as per offline convo, setting the whole config through constructor